src/CVSS/cvss.c

```diff
@@ -80,7 +80,7 @@ static struct cvss_metrics **cvss_impact_metricsptr(struct cvss_impact* impact,
     }
 }
 
-struct cvss_impact *cvss_impact_new(void) { return calloc(1, sizeof(struct cvss_metrics)); }
+struct cvss_impact *cvss_impact_new(void) { return calloc(1, sizeof(struct cvss_impact)); }
```

Comment:
heh, it pays off to use standard C functions instead of custom allocators, right? :-)

Reply:
@mpreisler Yes, I also have some reports about incorrect usage of `realloc`. I'm going to fix them in some of next pull requests.